lib/Capabilities.php

```diff
@@ -33,15 +38,28 @@
  */
 class Capabilities implements ICapability {
 	private IAppManager $appManager;
+	private LoggerInterface $logger;
+	private IConfig $config;
 
-	public function __construct(IAppManager $appManager) {
+	public function __construct(IAppManager $appManager, LoggerInterface $logger, IConfig $config) {
 		$this->appManager = $appManager;
+		$this->logger = $logger;
+		$this->config = $config;
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	public function getCapabilities() {
+	public function getCapabilities(): array {
+		$textColumnVariant = 'text-rich';
+		try {
+			if (version_compare($this->config->getSystemValueString('version', '0.0.0'), '26.0.0', '<')) {
+				$textColumnVariant = 'text-long';
+			}
+		} catch (NotFoundExceptionInterface|ContainerExceptionInterface $e) {
```

Comment:
this try-catch is also not necessary anymore as the exceptions would pop up only from autoloading.